### ISSUE TITLE
RTL form controls heights in Firefox

### ISSUE THREAD

**Issue by ffoodd** [Orange-OpenSource/Orange-Boosted-Bootstrap#626]
Our v5 form controls are way too tall in RTL —only in Firefox!?—, because of Helvetica Neue Arabic usage. See [our current RTL cheatsheet](https://v5-dev--boosted.netlify.app/docs/5.0/examples/cheatsheet-rtl/#overview).

In v4, this does not happen because our form controls have a `height`, see [v4.6 RTL Boostwatch](https://boosted.orange.com/docs/4.6/boostwatch/rtl-index#forms).

Tried `font-size-adjust` but it's inconsistent. We may need to set `height` again…

**Comment by ffoodd** [comment]
Just made v4 sandboxes on CodePen: [RTL input](https://codepen.io/ffoodd/pen/mdOmjWe) versus [LTR inputs](https://codepen.io/ffoodd/pen/rNWmrZo).

v5 sandboxes can't work for now because I missed to publish font files in our package. -_-

**Comment by ffoodd** [comment]
So FWIW, Helvetica Neue Arabic's baseline is higher: what's obvious in `input`s is visible for any text. Ascenders might be cut by an incorrect `line-height`.

So either there's an easy workaround (don't think so) or each vertical dimension needs to be customized in RTL—starting by adjusting `line-height`, then `padding` and `margin`… almost everywhere!

**Comment by ffoodd** [comment]
@Nurovek When you'll get there at some point, the Arabic Web UI Kit has been released on Zeplin and Sketch :tada:

**Comment by yannickcornaille** [comment]
This behavior is also true in Safari (so not only in Firefox).
There is a good explanation here : https://www.456bereastreet.com/archive/201108/line-height_in_input_fields/

I agree with @ffoodd's comment https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/626#issuecomment-781233578
I don't think there is an easy workaround either, it would be necessary to customize each `height`, `padding` and `margin` in RTL.
The only simple workaround is to change the `font-family`.

**Comment by ffoodd** [comment]
No, setting `height` makes more sense and was working fine in v4.

BTW there are several issues regarding form controls height in Bootsrtap v5 too, so `height` might be reintroduced on their side.

**Comment by MaxLardenois** [comment]
See #3818
